assets/src/edit-story/components/panels/document/slug/slug.js

```diff
@@ -68,9 +88,7 @@ function SlugPanel() {
     [updateStory]
   );
 
-  const handleChange = useCallback((value) => updateSlug(value, true), [
-    updateSlug,
-  ]);
+  const handleChange = useCallback((evt) => setSlug(evt.target.value), []);
```

Comment:
This feels strange. Should this function be cleaning the slug as it updates? Another option would be to display the formatted slug in the link 🤔 

![spaces](https://user-images.githubusercontent.com/22185279/111375108-7fdce780-8663-11eb-95cd-5b1eef3907cd.gif)

Reply:
Yep, the formatted slug should be used here, `cleanForSlug` with `isEditing` flag can be used for that.